generated/nifake/nifake.proto

```diff
@@ -96,7 +103,7 @@ enum NiFakeAttribute {
   NIFAKE_ATTRIBUTE_READ_WRITE_STRING = 1000002;
   NIFAKE_ATTRIBUTE_READ_WRITE_COLOR = 1000003;
   NIFAKE_ATTRIBUTE_READ_WRITE_INTEGER = 1000004;
-  NIFAKE_ATTRIBUTE_FLOAT_ENUM = 1000005;
+  NIFAKE_ATTRIBUTE_FLOAT_ENUM_NAME_OVERRIDE = 1000005;
```

Comment:
There's going to be several "breaking" changes to ni-fake that was done to test out things in hapigen. Since we only use ni-fake for Unit and Integration tests and don't publish / add the service to the server executable these should be fine.